### PR TITLE
https certificate validation missing on Netapp E-series hostgroup module

### PR DESCRIPTION
**Pull request by ghost**
##### SUMMARY

the "validate_certs" was type dict instead of bool and added the variable "certs" to use for the api calls. Tested with hostgroup creation/deletion

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME
netapp_e_hostgroup.py

##### ANSIBLE VERSION

```
ansible 2.3.1.0
  config file = /usr/local/git_tree/storage/ansible/ansible.cfg
  configured module search path = [u'/usr/local/git_tree/storage/ansible/modules']
  python version = 2.7.5 (default, Nov  6 2016, 00:28:07) [GCC 4.8.5 20150623 (Red Hat 4.8.5-11)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by lmprice** [comment]
@labazza I have a patch I'm about to push upstream that will resolve this issue. Overall it doesn't matter to me which one resolves the issue, but it looks like this patch is overwriting recent changes.

**Comment by ghost** [comment]
hi @lmprice, ah yes cause i worked on the module we use in prod which is the one available in ansible 2.3. the same issue exists also in the lun_mapping module.

**Comment by lmprice** [comment]
@labazza Whoops, I got confused, my patch is for resolving the issue in the mapping module. If you can rebase on the latest upstream, we can go ahead and utilize this patch for resolving the issue.

**Comment by ghost** [comment]
hi @lmprice i have updated the pull request with the upstream hostgroup module and discarded the lun_mapping patched by you. i have locally tested the module with present/update/absent values.

**Comment by ghost** [comment]
authentication module has the same issue.

**Comment by lmprice** [comment]
@labazza I'm working on a patch for the auth module. I'm going to test your changes tomorrow and get back to you on the review. Sorry for the delay.

**Comment by ghost** [comment]
ok no worries @lmprice i have also created a module for setting the system name. it has been tested with e2800 and e5600

**Comment by lmprice** [comment]
@labazza The maintainers will require that you split the changes into one PR per module before they are committed (or at least that's what I recall). I do have some comments on that new module, but I'll wait for the new PR.

**Comment by ghost** [comment]
@lmprice i have removed the netapp_e_name module from this pull request

**Review by lmprice — APPROVED** [review]
This looks good to me. I would ask that you submit the other commits (aforementioned), as a separate patch. 
shipit

**Comment by lmprice** [comment]
shipit

File: lib/ansible/modules/storage/netapp/netapp_e_hostgroup.py [hunk 1]
```diff
@@ -317,7 +317,7 @@ def main():
             hosts=dict(required=False, type='list'),
             api_url=dict(required=True),
             api_username=dict(required=True),
-            validate_certs=dict(required=False, default=True),
+            validate_certs=dict(type='bool'),
```
Comment (Akasurde) [inline lib/ansible/modules/storage/netapp/netapp_e_hostgroup.py]: set default to True.

**Comment by Akasurde** [comment]
@labazza Are you still working on this ?

**Comment by ghost** [comment]
@Akasurde  this is ready since when i commented on Aug 28, 2017

**Comment by Akasurde** [comment]
@labazza Could you please address code review comment ?

**Comment by ghost** [comment]
Pushed . thanks

**Comment by Akasurde** [comment]
@labazza Thanks. There is an issue in CI run - 

```
2018-05-24 10:48:12 ERROR: Found 1 validate-modules issue(s) which need to be resolved:
2018-05-24 10:48:12 ERROR: lib/ansible/modules/storage/netapp/netapp_e_hostgroup.py:0:0: E325 argument_spec for "validate_certs" defines type="bool" but documentation does not (75%)
```

**Pull request closed without merging**